contracts/program/ProgramImplementation.sol

```diff
@@ -74,10 +85,17 @@ contract ProgramImplementation is AccessControlEnumerable, Initializable {
     }
   }
 
-  // @notice Update metaPtr (only by PROGRAM_OPERATOR_ROLE)
+  /// @notice Update metaPtr (only by PROGRAM_OPERATOR_ROLE)
   /// @param newMetaPtr new metaPtr
   function updateMetaPtr(MetaPtr memory newMetaPtr) external onlyRole(PROGRAM_OPERATOR_ROLE) {
     emit MetaPtrUpdated(metaPtr, newMetaPtr);
     metaPtr = newMetaPtr;
   }
+
+  /// @notice Create a new round (only by PROGRAM_OPERATOR_ROLE)
+  /// @param encodedParameters Encoded parameters for round creation
+  function createRound(bytes calldata encodedParameters) external onlyRole(PROGRAM_OPERATOR_ROLE) {
+    address round = IRoundFactory.create(encodedParameters);
+    emit RoundCreated(round, msg.sender);
```

Comment:
I don't know if this is an overkill but the goal was to emit the operator address when the round is created. In theory this info is already available on-chain so in 2 minds on this